### PR TITLE
Add Carrion Wiki.gg

### PR DESCRIPTION
**Pull request by DrWhoFan13**
I added Carrion wiki.gg, which I (anonymously) requested yesterday. Considering that the Fandom wiki has _39_ pages, I have no doubt that the fork will go through.

### CODE REVIEW

File: data/sitesEN.json [hunk 1]
```diff
@@ -885,6 +885,28 @@
       "official"
     ]
   },
+  {
+    "id": "en-carrion",
+    "origins_label": "Carrion Fandom Wiki",
+    "origins":[
+      {
+      "origin": "Carrion Fandom Wiki",
+      "origin_base_url": "https://carrion.fandom.com",
```
Comment (KevinPayravi) [inline data/sitesEN.json:894]: Please remove the `https://`

**Review by KevinPayravi — CHANGES_REQUESTED** [review]
Thanks for the submission! One small fix requested.

**Comment by DrWhoFan13** [comment]
Removed the `https://` as requested.

**Review by KevinPayravi — APPROVED** [review]
Thank you!

**Pull request merged**